src/plugins/vis_type_timeseries/public/application/components/vis_types/timeseries/vis.js

```diff
@@ -212,7 +212,6 @@ export class TimeseriesVisualization extends Component {
           seriesDataRow.groupId = groupId;
           seriesDataRow.yScaleType = yScaleType;
           seriesDataRow.hideInLegend = Boolean(seriesGroup.hide_in_legend);
-          seriesDataRow.useDefaultGroupDomain = !isCustomDomain;
```

Comment:
Could we post the behavior, for before / after change to ensure we are getting expected behavior ?

Reply:
@kavilla @mihirsoni @joshuarrrr @ananzh 
**This is how the Y-axis was looking due to the bug.**
![Screen Shot 2022-08-09 at 11 15 22 AM](https://user-images.githubusercontent.com/62020972/183730647-db615139-1722-4a43-b9de-62e461341dca.png)

But in my investigation I came to know that, this was happening because of a prop `useDefaultGroupDomain`. Actually this prop was introduced into the elastic charts to allow clustering for the multiple series which will share a common Y domain i.e for example the stacked and non- stacked bars will have the same Y common domain/axis. As per the elastic/charts whenever the `useDefaultGroupDomain` is set to true, it will force to group the series/visualiations groupId's to a  default group domain. But, TSVB is not a shared Y domain. So, I felt `useDefaultGroupDomain` may not be the right prop for TSVB. 

Either we can remove the line 215 as I did above or by setting the `useDefaultGroupDomain` to **false** is also solving this issue.

**After removing the line, the output is looking like:**
![Screen Shot 2022-08-09 at 11 34 45 AM](https://user-images.githubusercontent.com/62020972/183735404-90b496cf-92f5-497b-9da1-70ff7e808d62.png)

**or, after setting the `useDefaultGroupDomain` to false**
![Screen Shot 2022-08-09 at 11 36 50 AM](https://user-images.githubusercontent.com/62020972/183735767-a73351d0-d992-4a14-9308-d27ebc5e8d94.png)

As,@joshuarrrr asked about the tests, i could think of like to verify whether the `useDefaultGroupDomain` is set to false so that TSVB will work as expected.
This is what I have with me until now. Please correct me if i am wrong anywhere.